libraries _PLUGIN145 ITHO FAN/Itho/IthoCC1101.cpp

```diff
@@ -344,25 +341,26 @@ void IthoCC1101::initReceive()
 void  IthoCC1101::initReceiveMessage2(IthoMessageType expectedMessageType)
 {
 	uint8_t marcState;
-	
+
 	writeCommand(CC1101_SIDLE);	//idle
-	
-	//set datarate	
-	writeRegister(CC1101_MDMCFG4 ,0x5A); // set kBaud
+
+	//set datarate
+	writeRegister(CC1101_MDMCFG4 ,0x9A); // set kBaud
```

Comment:
If this is a baudrate setting, shouldn't it be a configurable parameter?
I can't imagine why it would have worked before and should now be using a different value, unless there are several revisions of the same device.
Then it should have a configurable parameter

Reply:
The previous plugin was based on a slightly dated Itho library. I have included the latest changes made by supersjimmie on the non plug-in version of the Itho library: https://github.com/supersjimmie/IthoEcoFanRFT/commit/7ae2c9e34e13a628988caa28f975c9929fb9676d

I added a comment on this.